src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1422,3 +1422,27 @@ def _configure_security_policy_storage_params(arg_ctx):
 
         c.argument('allow_data_loss',
                    arg_type=allow_data_loss_param_type)
+
+    ###################################################
+    #             sql sensitivity classification      #
+    ###################################################
+    with self.argument_context('sql db classification update') as c:
+        c.argument('schema_name',
+                   required=True,
+                   help='The name of the schema.')
+
+        c.argument('table_name',
+                   required=True,
+                   help='The name of the table.')
+
+        c.argument('column_name',
+                   required=True,
+                   help='The name of the column.')
+
+        c.argument('information_type',
```

Comment:
Can the information_type be any string or only some allowed values?

Reply:
The allowed values are set in the information protection policy and can be changed. There's a check if the value passed exists in the policy.